pegasus/helpers/page_helpers.rb

```diff
@@ -88,3 +88,35 @@ def youtube_embed(youtube_url)
 
   %Q{<iframe title="YouTube video player" width="250" height="141" src="http://www.youtube.com/embed/#{youtube_id}" frameborder="0" allowfullscreen></iframe>}.html_safe
 end
+
+# A hacky method to generate locale-aware links to our lesson plans. This is
+# hacky primarily because we are forced to hardcode here a list of supported
+# languages; a list which we have no guarantee will remain accurate in the long
+# term.
+def hacky_localized_lesson_plan_url(path)
+  # This is the list of languages currently supported by the CurriculumBuilder
+  # i18n sync. This list should be kept in sync with the LANGUAGES settings
+  # variable in the curriculumbuilder project. (curriculumBuilder/settings.py)
+  curriculumbuilder_languages = [
+    'es-mx', 'es-es', 'hi-in', 'it-it', 'pl-pl', 'sk-sk', 'th-th'
+  ]
+
+  # This is a list of additional languages we want to support. These are
+  # languages for which there does exist content in that language on
+  # curriculum.code.org, but which aren't regularly synced.
```

Comment:
out of curiosity, why aren't these languages regularly synced?

Reply:
the CurriculumBuilder sync has historically been _very_ slow, specifically because of the step where we publish the translated results to S3, so we've significantly restricted the number of languages which get included in that sync.

We've recently done a bunch of work to make the sync faster, and hope to be able to expand that set of languages in the near future. But even then I suspect that our eyes will remain larger than our stomach, as it were.

Reply:
haha you're probably right 😜  thanks for that context!

Reply:
How do you add a language-URL path in Curriculum Builder without adding it to the sync? (e.g. ar-sa)

Reply:
By doing a one-off sync